Phases/Assessments/Phase_3.md

```diff
@@ -0,0 +1,44 @@
+# Phase 3 Assessment
+
+The phase 3 assessment will test your ability to combine all of your foundational skills to build real, full-stack web applications both alone and in small teams.
+
+Note that the phase 3 assessment differs significantly from the previous assessments in that you will be **pair programming as part of your evaluation interview**. A key skill to learn in phase 3 is how to build code in team, and the interviewer will be assessing your ability to work with them to build out a feature.
+
+## Skills Assessed
+
+How to combine all of your HTML and CSS, HTTP Server and Browser skills together
+More JavaScript design patterns
+Full Web Application Architecture
+How Authentication works on most of the web
+Persisting web application state in a SQL database
+More advances asynchronous patters in JavaScript
+How to use Git and GitHub on a team
+How to render dynamically generated HTML
+
+- HTTP / Web Applications (Intermediate Level)
+- HTML & CSS (Intermediate Level)
+- HTTP (Intermediate Level)
```

Comment:
I added the `/ Web` part, but it probably reduces clarity. Will remove. I'm following the language defined in the v1.5 curriculum repo